### ISSUE TITLE
It would be helpful if a guide for running the binaries was included

### ISSUE THREAD

**Issue by Suryced** [skni-kod/MicrOS#105]
I tried playing around with MicrOS but was stuck getting the error: 
/opt/cross/i386-elf/bin/ar: error while loading shared libraries: libfl.so.2: cannot open shared object file: No such file or directory
make: *** [makefile:31: bin/libc.a] Error 127
I followed the guide on installing the OS with configure.sh. I ran build.sh and make but certain shared object files couldn't be found. I see that nobody has touched this repository for more than a year.

**Comment by MatiF100** [comment]
Were you compiling it using WSL or on GNU/Linux?

**Comment by MatiF100** [comment]
Also, if you only want to run the system and see how it is working, you do not need to compile it. There is floppy disk image under Releases tab, you can run in QEMU

**Comment by MinusPL** [comment]
@Suryced Yes, sourcecode provided here is mostly for referencing how it works, not really for building. We don't have (yet) whole set of tools needed for building released. Cannot also disclose when or if at all we're going to release them though.

Currently your best choice for testing MicrOS is to go to releases and grab newest one which at the moment is 1.1.
Just remember that 1.1 is still very unstable and works mostly on qemu. We have finished works on 1.2 version not long ago and currently we're trying to gather everything and prepare official 1.2 release.

Sorry we can't give you tools required to build OS, we do hope you'll be interested in future releases.
